### DISCUSSION TITLE
How to Install/Update Built-In Extensions on Windows when installing manually (w/o automatic.sh install)

### DISCUSSION THREAD

**Discussion by mgreed** [vladmandic/sdnext#89] [Q&A] [answered]
I manually installed this fork on Win10, with git clone and pip -r. I'm able to launch using webui.bat, but it looks like a lot of the built-in extensions aren't installed. If I check for updates from the extensions tab, the progress bar flashes for a second, but nothing else changes. The Version and Update columns are/remain empty, even for extensions that appear to be installed and working (eg. LoRA).

Empty extension folders:
sd-*
seed_travel
stable-diffusion-*

Is there a way to get those built-ins installed correctly? Is there additional work that needs to be done for a successful manual install on Windows (without WSL/Bash)?

**Comment by vladmandic** [comment, marked as answer]
for manual initialization of built-in extensions, you can try `git submodule update --init --recursive`
new installer/launcher is on the way and should be cross-platform. see #92 for details.

**Reply by mgreed** [reply]
That did it, thank you!
